MyBooking/Views/BookingView/src/App.js

```diff
@@ -235,13 +328,52 @@ class App extends React.Component {
                                             adverts={this.state.searchResult}
                                             searchByAddress={this.handleAddressesSearch}
                                             selectedAddress={this.state.selectedAddress}
```

Comment:
`selectedAddress` is the address to search, or what the selected means?

Reply:
It means that there will be text, that user input in the search field

Reply:
Maybe you can pick a name for the variable, which will mean that it is the entered text or text to search for?

Reply:
so, why searchAddress is bad?

Reply:
I'm talking about the selectedAddress.
We have searchByddverss and selectedAddress whether they should be named so that it is clear that the search result is related to the address?

Reply:
searchAddress  it's good

Reply:
let me fix it in the next pr

Reply:
+